schema.graphql

```diff
@@ -861,6 +862,14 @@ input ToggleUserTagAssignInput {
   userId: ID!
 }
 
+type TransactionLog {
+  createdBy: String!
+  message: String!
+  model: String!
+  timestamp: DateTime!
+  type: String!
+}
+
```

Comment:
Why are we adding `TransactionLog` type to GraphQL schema? Logging is an operation specific to the backend and shouldn't be exposed to clients. Thus, there is no point adding in the schema.